Automattic/wp-calypso
# Jetpack section: add Automated Transfer step for Business sites

#### Changes proposed in this Pull Request

* Show non-Atomic Business sites a CTA to Automated Transfer.

Fixes 1179060693083348-as-1179702911174270

#### Questions

* What should we do with eCommerce sites? Should they have direct access to Backup and Jetpack?

#### Testing instructions

**Prerequisite:** non-Atomic WPCOM site with Business plan.
* Run this PR and enable the `jetpack/features-section`.
* Visit Jetpack > Backup.
* Verify that you see the Automated Transfer CTA and that it matches the design.
* Visit Jetpack > Scan.
* Verify that you see the Automated Transfer CTA and that it matches the design.
* Click the CTA to initiate the Automated Transfer process.
* Verify that the CTA display a loading indicator and it is disabled while the AT process is in progress.

#### Demo 
##### While trying to access the Jetpack Backup section
![image](https://user-images.githubusercontent.com/3418513/84782823-e5aa1f00-afbe-11ea-8d89-b08dcd5e3fec.png)

##### While trying to access the Jetpack Scan section
![image](https://user-images.githubusercontent.com/3418513/84782853-f22e7780-afbe-11ea-814d-51754ad0a768.png)
